src/Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -122,12 +123,14 @@ public void MergeInto(TextFileConfiguration destination)
         /// </summary>
         /// <param name="key">Name of the argument (excluding the dash prefix).</param>
         /// <returns>Values for the specified argument.</returns>
-        public string[] GetAll(string key)
+        public string[] GetAll(string key, ILogger logger = null)
         {
             // Get the values with the - prefix.
             if (!this.args.TryGetValue($"-{key}", out List<string> values))
                 values = new List<string>();
 
+            logger?.LogDebug("GetAll('{0}') returned {1} entries.", values.Count);
```

Comment:
Maybe change message to "{0} entries were returned for the key '{1}'"?

Reply:
Done.